package.json

```diff
@@ -4,10 +4,10 @@
   "author": "Google Inc.",
   "description": "Google APIs Authentication Client Library for Node.js",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=4"
```

Comment:
We weren't testing on on 0.10 ... so it's hard to say what was real here.  What do you think we should do?  
- [ ] Keep this at 0.10 and add 0.10 back to travis config
- [ ] Move this to 4, and bump semver major
- [ ] Move this to 4, and don't bump semver major since technically 0.10 has been out of support a loooooooong time

I'm cool with any of these :)

Reply:
Ok, so I tested 0.10 to see if it even worked.  Result: 
![image](https://user-images.githubusercontent.com/534619/32567203-6ec53970-c46f-11e7-8039-c1e171fe24dc.png)

There's no way anyone was actually using this - it fails on npm install, with about 100 peer dependency failures.  I vote we keep supported at 4, and don't bother with the semver major bump.  I don't know how it's possible that this is breaking someone who wasn't already broken.

Reply:
Doubling back.  As part of the PR, we're now changing the way the module is imported/exported, so it's a semver major bump anyway :)